projects/storefrontlib/src/cms-components/checkout/components/shipping-address/address-form/address-form.component.ts

```diff
@@ -172,6 +168,19 @@ export class AddressFormComponent implements OnInit, OnDestroy {
     this.addresses$ = this.userAddressService.getAddresses();
   }
 
+  getTitles(): Observable<Title[]> {
+    return combineLatest([
+      this.translation.translate('addressForm.noneTitle'),
+      this.userService.getTitles(),
```

Comment:
'getTitles' is deprecated you have to use new lib `@spartacus/user` and new method from facade of user profile

Reply:
He will revert back to his old one and use the deprecated method.

We shouldn't create such dependencies from our main lib with feature libs.

Also, the one who will move this to the feature-lib in the near future will update the dependencies accordingly.